Arrowgene.Ddon.Rpc.Web/Route/KickRoute.cs

```diff
@@ -0,0 +1,54 @@
+using Arrowgene.Ddon.Database.Model;
+using Arrowgene.Ddon.GameServer;
+using Arrowgene.Ddon.Rpc.Command;
+
+namespace Arrowgene.Ddon.Rpc.Web.Route
+{
+    public class KickRequest
+    {
+        public KickRequest()
+        {
+            AccountName = string.Empty;
+        }
+
+        public string AccountName { get; set; }
+
+    }
+
+    public class KickCommand : RpcPostCommand<KickRequest>
+    {
+        public KickCommand(KickRequest entry) : base(entry)
+        {
+        }
+
+        public override string Name => "KickCommand";
+
+        public override RpcCommandResult Execute(DdonGameServer gameServer)
+        {
+            var accountName = _entry.AccountName.ToLowerInvariant();
+            bool action = false;
+            foreach (var client in gameServer.ClientLookup.GetAll())
+            {
+                if (client.Account.NormalName == accountName)
+                {
+                    client.Close();
+                    action = true;
+                    break;
+                }
+            }
+            Account account = gameServer.Database.SelectAccountByName(accountName);
+            action = account != null && gameServer.Database.DeleteConnection(gameServer.Id, account.Id) || action;
```

Comment:
How does this handle the case where the player isn't connected to the queried server? I feel it should return an HTTP error code